src/simulator_core/entities/assets/heatexchanger.py

```diff
@@ -0,0 +1,60 @@
+#  Copyright (c) 2023. Deltares & TNO
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Heat exchanger classes."""
+from pandapipes import create_heat_exchanger, pandapipesNet
+
+from simulator_core.entities.assets.junction import Junction
+
+
+class HeatExchanger:
```

Comment:
If you inherit from assetabstract you do not need to instantiate the self.from_junction and asset name etc.